libs/application/api/core/src/lib/application/application.service.ts

```diff
@@ -111,6 +115,58 @@ export class ApplicationService {
     })
   }
 
+  async findAllByInstitutionAndFilters(
+    nationalId: string,
+    page: number,
+    count: number,
+    status?: string,
+    applicantNationalId?: string,
+    from?: string,
+    to?: string,
+  ): Promise<ApplicationPaginatedResponse> {
+    const statuses = status?.split(',')
+    const typeIds = this.getTypeIdsForInstitution(nationalId)
+    const toDate = to ? endOfDay(new Date(to)) : undefined
```

Comment:
No option to filter by a specific Time?

Reply:
I was trying to mimic the admin portal where to date is inclusive but you're right, should be done client side and leave the posibility of a specific time filter from the api